src/qmeta/QMeta.h

```diff
@@ -35,6 +35,11 @@
 #include "qmeta/QStats.h"
 #include "qmeta/types.h"
 
+namespace lsst::qserv::qdisp {
+    class MessageStore;
+    class QueryMessage;
```

Comment:
According to @n8pease, spaces aren't allowed in the forward declarations.

Reply:
auto formatter